components/molecules/ContributorTable/contributor-table.tsx

```diff
@@ -24,12 +24,14 @@ interface CardTableProps {
 const ContributorTable = ({ contributor, topic, repositories, limit, isHoverCard }: CardTableProps): JSX.Element => {
   const { data, isLoading } = useTopicContributorPRs(contributor, topic, repositories, limit);
 
-  return data.length > 0 ? (
+  const sortedPr = data?.sort((a: any, b: any) => parseInt(b.updated_at) - parseInt(a.updated_at));
+
+  return sortedPr.length > 0 ? (
     <>
       <div className="flex flex-col">
         <LatestPrTableHeader isHoverCard={isHoverCard} />
         <div className="flex flex-col gap-0.5">
-          {data.map(
+          {sortedPr.map(
```

Comment:
As a note, `sort()` mutates the original array, so a separate `sortedPr` variable isn't needed

Reply:
If you want to use a new variable, you can create a copy of the original array using spread syntax or the `slice()` method on the  array